fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/impl/content/registry/LegacyFlammableBlockRegistryImpl.java

```diff
@@ -32,9 +32,10 @@
 import net.fabricmc.fabric.api.resource.ResourceReloadListenerKeys;
 import net.fabricmc.fabric.api.resource.SimpleSynchronousResourceReloadListener;
 
-public class FlammableBlockRegistryImpl implements FlammableBlockRegistry, SimpleSynchronousResourceReloadListener {
+@Deprecated
+public class LegacyFlammableBlockRegistryImpl implements FlammableBlockRegistry, SimpleSynchronousResourceReloadListener {
 	private static final FlammableBlockRegistry.Entry REMOVED = new FlammableBlockRegistry.Entry(0, 0);
-	private static final Map<Block, FlammableBlockRegistryImpl> REGISTRIES = new HashMap<>();
+	private static final Map<Block, LegacyFlammableBlockRegistryImpl> REGISTRIES = new HashMap<>();
 	private static final Collection<Identifier> RELOAD_DEPS = Collections.singletonList(ResourceReloadListenerKeys.TAGS);
 	private static int idCounter = 0;
```

Comment:
Do we need all this? Ideally the new api should handle the implimentation

Reply:
wouldn't it sorta break old API to add a new dependency? Also the new API removes one of the functions this supports (explanation in PR description), so forwarding the full implementation isn't really possible. I don't see this as a big issue because these are not high-conflict areas and the two APIs can work together...the new one is just nicer

Reply:
It won't break the old api if you bump the version of the old module and add the new module as a depedency, because the descriptors of stuff in api package remain the same and binary compatibility remains.

Reply:
That doesn't change the fact that changing it to use the new API wouldn't really work due to it not implementing the .clear method.

Reply:
Also, the new implementation works differently in that it preserves add order when loading, so this change in behavior has potential to cause unwanted side effects if hooked up to old api. Best to keep them separate.